src/test/groovy/com/ofg/uptodate/finder/JCenterReponseProvider.groovy

```diff
@@ -1,11 +1,12 @@
 package com.ofg.uptodate.finder
-
-import static com.github.tomakehurst.wiremock.client.WireMock.aResponse
-import static com.github.tomakehurst.wiremock.client.WireMock.get
-import static com.github.tomakehurst.wiremock.client.WireMock.urlMatching
+import static com.github.tomakehurst.wiremock.client.WireMock.*
 
 class JCenterReponseProvider {
     void stubInteractionForJcenter(String group, String name, String response) {
         stubInteraction(get(urlMatching("^.*${group.split('\\.').join('/')}/${name}/maven-metadata.xml")), aResponse().withHeader('content-type', 'application/xml').withStatus(200).withBody(response))
     }
+
+    void stubProxyInteractionForJcenter(String group, String name, String response) {
+        stubProxyInteraction(get(urlMatching("^.*${group.split('\\.').join('/')}/${name}/maven-metadata.xml")), aResponse().withHeader('content-type', 'application/xml').withStatus(200).withBody(response))
```

Comment:
you're duplicating part of the code in each of the methods - maybe we can extract this part ?